Sources/LLVM/IRBuilder.swift

```diff
@@ -1090,14 +1090,36 @@ extension IRBuilder {
   ///
   /// - returns: A value representing the result of a load from the given
   ///   pointer value.
+  @available(*, deprecated, message: "Use buildLoad(type:ptr:ordering:volatile:alignment:name) instead")
   public func buildLoad(_ ptr: IRValue, ordering: AtomicOrdering = .notAtomic, volatile: Bool = false, alignment: Alignment = .zero, name: String = "") -> IRInstruction {
     let loadInst = LLVMBuildLoad(llvm, ptr.asLLVM(), name)!
     LLVMSetOrdering(loadInst, ordering.llvm)
     LLVMSetVolatile(loadInst, volatile.llvm)
     LLVMSetAlignment(loadInst, alignment.rawValue)
     return loadInst
   }
-
+  
+  /// Build a load instruction that loads a value from the location in the
+  /// given value.
+  ///
+  /// - parameter type: The type of value to load.
```

Comment:
I think we ought to be clearer that this is the type of the pointed-to value and not the type of the pointer value.  Maybe

```diff
+ - parameter type: The type of the value loaded from the given pointer.
```